packages/user-ui/src/pages/project/controller.rs

```diff
@@ -196,27 +201,36 @@ impl Controller {
     }
 }
 
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct SurveyResponses {
```

Comment:
design and use api_model.

Reply:
@hackartists 
This part is a model built to display data from the UI regardless of the API. 
Therefore, it is not related to the apimodel, so it is not configured separately in the model directory.

Reply:
I think it would be gotten from API, right?
If right, why doesn't it be api_model?

Reply:
The model obtained from the API will be the corresponding part of DeliberationResponse, and
SurveyResponse is a model for parsing into a structure suitable for the UI and displaying data in the form of a chart on the UI.
Therefore, since the model itself is not a model obtained from the API, it was not placed as api_model.